lib/dom_utils.coffee

```diff
@@ -167,15 +167,20 @@ DomUtils =
       node = node.parentNode
     false
 
-  # True if element contains the active selection range.
+  # True if element contains the active selection range, or if the element does not support its selection
+  # being accessed.
   isSelected: (element) ->
     if element.isContentEditable
-      node = document.getSelection()?.anchorNode
+      node = document.getSelection().anchorNode
       node and @isDOMDescendant element, node
     else
-      # Note.  This makes the wrong decision if the user has placed the caret at the start of element.  We
-      # cannot distinguish that case from the user having made no selection.
-      element.selectionStart? and element.selectionEnd? and element.selectionEnd != 0
+      try
+        element.selectionStart != element.selectionEnd
```

Comment:
I liked that feature too, but that's not what the comment above the function said it was doing. This function needs a new name and description if that's what it should be doing.